css-backgrounds-4/Overview.bs

```diff
@@ -242,8 +242,70 @@ Painting Area: the 'background-clip' property</h3>
 <h2 id="corners">
 Corners</h2>
 
-<h3 id=corner-sizing>
-Corner Sizing: the 'border-radius' property</h3>
+<h3 id="corner-sizing">
+Corner Sizing: the 'border-radius' and 'border-*-radius' shorthand properties</h3>
+
+<h4 id="physical-corner-sizing">
+Physical Corner Sizing:
+The 'border-top-radius', 'border-right-radius',
+'border-bottom-radius', 'border-left-radius' side shorthands</h4>
+
+	<pre class=propdef>
+		Name: border-top-radius, border-right-radius, border-bottom-radius, border-left-radius
+		Value: <<length-percentage [0,&infin;]>>{1,2} [ / <<length-percentage [0,&infin;]>>{1,2} ]?
+		Initial: 0
+		Applies to: all elements (but see prose)
+		Inherited: no
+		Percentages: Refer to corresponding dimension of the <a>border box</a>.
+		Computed value: see individual properties
+		Animatable: see individual properties
+	</pre>
+
+	<p>The 'border-*-radius' shorthands set the two 'border-*-*-radius'
+	longhand properties of the related side. If values are given before
+	and after the slash, then the values before the slash set the
+	horizontal radius and the values after the slash set the vertical radius.
+	If there is no slash, then the values set both radii equally.
+	The two values for the radii are given in the order
+	top-left, top-right for 'border-top-radius',
+	top-right, bottom-right for 'border-right-radius'
+	bottom-left, bottom-right for 'border-bottom-radius',
```

Comment:
I'm not sure about this, would it be better to follow the same order as in `border-radius`?

So `border-radius: 1px 2px 3px 4px` would be equivalent to

```css
border-top-radius: 1px 2px;
border-bottom-radius: 3px 4px;
```
instead of
```css
border-top-radius: 1px 2px;
border-bottom-radius: 4px 3px;
```

Reply:
I believe it makes more sense to keep them from top to bottom and left to right, especially in regard of their logical counterparts. E.g. in horizontal writing modes you probably would not want the values switch their order depending on whether they are left-to-right or right-to-left.

Though I created an issue in the spec. so we can discuss this separately.